v3/cmd/zlint/main.go

```diff
@@ -32,11 +32,14 @@ import (
 	"github.com/zmap/zlint/v3"
 	"github.com/zmap/zlint/v3/formattedoutput"
 	"github.com/zmap/zlint/v3/lint"
+
+	_ "github.com/zmap/zlint/v3/profiles"
```

Comment:
We need to trigger the `init` functions in `/profiles` so that we can build our `FilterOptions` appropriately. If we try to add this import into `lint` then it causes an import cycle between `lint` and `profiles` :confused: